site/docs/src/json/tenants/request.json

```diff
@@ -134,6 +134,7 @@
         "type": "randomBytes"
       },
       "registrationVerificationIdTimeToLiveInSeconds": 86400,
+      "samlv2AuthNRequestIdTimeToLiveInSeconds": 360,
```

Comment:
This defaults to `300`.

Reply:
I figured we would want a non-default value here to demonstrate that usage. But I will adjust it.